Gruntfile.js

```diff
@@ -391,7 +391,7 @@ module.exports = function(grunt) {
           .map(module =>
             [
               `cd ${module}`,
-              `rm -rf node_modules`,
+              `npm dedupe`,
```

Comment:
Running `dedupe` when there are no `node_modules` ends up generating an empty `package-lock`, so the next `ci` errors so beautifully:
```
TypeError: Cannot read property '@medic/bulk-docs-utils' of undefined at Object.keys.forEach.name 
(/home/diana/.nvm/versions/node/v11.15.0/lib/node_modules/npm/node_modules/lock-verify/index.js:27:40)
```
`npm ci` already removes `node_modules` as its first action.